<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookWindow</class>
 <widget class="QWidget" name="AddressBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>1000</height>
      </size>
     </property>
     <widget class="QWidget" name="page_function">
      <layout class="QVBoxLayout" name="verticalLayout_function">
       <item>
        <widget class="QPushButton" name="pushButton_addressbook_manager">
         <property name="text">
          <string>通讯录管理器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_friend_inform">
         <property name="text">
          <string>好友通知</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_group_inform">
         <property name="text">
          <string>群聊通知</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_friend_list">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>400</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_friend_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>298</width>
            <height>697</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_friend_list"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_friendlist">
      <widget class="QListWidget" name="listWidget_friend_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>291</width>
         <height>841</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_find">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_find_friend">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_find_friend">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>50</horstretch>
          <verstretch>25</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add_friend">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="stackedWidgetPage_friend_inform">
      <layout class="QVBoxLayout" name="verticalLayout_friend_request">
       <item>
        <widget class="QScrollArea" name="scrollArea_friend_request">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>965</width>
            <height>826</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_request_list"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stackedWidgetPage_group_inform"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
